src/libostree/ostree-repo-commit.c

```diff
@@ -2858,16 +2858,6 @@ write_dfd_iter_to_mtree_internal (OstreeRepo                  *self,
       if (!glnx_fstatat (src_dfd_iter->fd, dent->d_name, &stbuf, AT_SYMLINK_NOFOLLOW, error))
         return FALSE;
 
-      const char *loose_checksum = devino_cache_lookup (self, modifier, stbuf.st_dev, stbuf.st_ino);
```

Comment:
I think this patch should be included with the last commit since they're not
*really* independent in the sense that I think we were doing this early lookup
intentionally before.  It was intending to be a fast path.